README.md

```diff
@@ -1,13 +1,83 @@
 
 dive deeper into your code
 
 [![hydrox logo](https://raw.githubusercontent.com/helpshift/hydrox/master/template/assets/img/big.png)](http://helpshift.github.io/hydrox)
 
 ## Installation
 
+### Deps Project:
+1. In your `deps.edn`, add `hydrox` and `fudje` (optional, for more readable tests) to the `dev` alias:
+```clojure
+  :dev      {:extra-deps
+             {parkside-securities/repl-bootstrap {:git/url "git@github.com:parkside-securities/repl-bootstrap.git"
+                                                  :sha     "4950c722bd306a06a5e51a1471889fd4bfcd6aa7"}
+              floatingpointio/graphql-builder    {:mvn/version "0.1.6"}
+              fudje                              {:mvn/version "0.9.7"}
+              parkside-securities/hydrox         {:mvn/version "0.10.36-SNAPSHOT"}}
+```
+2. Create the `:documentation` map in your `deps.edn`. This is how Hydrox knows where to put generated documents:
+```clojure
+:documentation {:site   "test-docs"
+                 :output "docs"
+                 :template {:path "template"
+                            :copy ["assets"]
+                            :defaults {:template     "article.html"
+                                       :navbar       [:file "partials/navbar.html"]
+                                       :dependencies [:file "partials/deps-web.html"]
+                                       :navigation   :navigation
+                                       :article      :article}}
+                 :paths ["test"]
+                 :files {"test-docs-name"
+                         {:input "test/your_app/docs_test.clj"
+                          :title "Your App Tests"
+                          :subtitle "change these for your app."}}}}
+
+```
+3. Create the `docs` folder in your project. Intialize with a `docs/.gitignore` that includes the following lines:
+```
+*
+*/
+!.gitignore
+# docs contents are generated by hydrox and copied to docker image at runtime 
+# generated artifacts shouldn't be checked into git in general.
+```
+
+4. Create the `template` folder and copy the templates and assets from this project into it so the documentation generator can access them and they will have a standardized look.
+
+5. Set up the server for your new files as follows:
+- If you do not already have the latest [pedestal](https://github.com/parkside-securities/pedestal) in your project, add it to deps.edn,
+  then add to your project according to the pedestal README.
+- In your `config.edn`, configure pedestal to include a `:file-path` like so:
+```clojure
+:pedestal/service {:service-map  #ig/ref :your-app.pedestal/service
+                   :interceptors [whatever the interceptors already were if you've got 'em']
+                   :dev?         #duct/env "PEDESTAL_SERVICE_DEV"
+                   :file-path    "docs"}
+```
+
+6. Open the repl with `clj -A:dev -e "(dev)(go)" -r`
+
+7. In the repl run:
```

Comment:
Please add a helper function in `dev.clj` of `repl-bootstrap` so that developers can generate the docs in one call. I also wonder if `dive` and `surface` can be mapped as `ig/init-key!` and `ig/halt-key`.